osu.Game/Overlays/BeatmapListing/BeatmapSearchMultipleSelectionFilterRow.cs

```diff
@@ -118,6 +119,8 @@ public MultipleSelectionFilterTabItem(T value)
                 });
             }
 
+            protected override HoverSounds CreateHoverSounds() => new HoverClickSounds(HoverSampleSet.TabSelect);
```

Comment:
I don't know how to explain why this is needed, but it **needs an inline comment**, or it needs to be removed.

Reply:
Added inline comment in the alternative branch.